api/controllers/WebActionsController.js

```diff
@@ -7,7 +7,9 @@
 var Promise = require('bluebird');
 var request = Promise.promisifyAll(require('request'));
 const Mixpanel = require('mixpanel');
+const PartnerService = require('../services/PartnerService');
```

Comment:
I don't think you need to `require` this. http://sailsjs.com/documentation/concepts/services